draft-ietf-oauth-device-flow.xml

```diff
@@ -259,12 +257,14 @@
 	<t>
 	  For example, the client makes the following HTTPS request (line
 	  breaks are for display purposes only):
+  </t>
+  <t>
 	  <figure><artwork><![CDATA[
-   POST /token HTTP/1.1
+   POST /device_authorization HTTP/1.1
```

Comment:
I'm intrigued by this particular change, as compared to its cousin ETSI standard ([ETSI TS 103 407](http://www.etsi.org/deliver/etsi_ts/103400_103499/103407/01.01.01_60/ts_103407v010101p.pdf)) the choice of URI paths appears to have evolved like so:

| Action | ETSI | IETF Draft 4 | This PR |
|--------|------|-------------|---------|
| Request user_code | POST /associate | POST /token | POST /device_authorization
| Device Auth | POST /token | POST /token | POST /token

As minor as it is, maybe having this endpoint match the ETSI equivalent might be a suitable idea?